inscriptions/ISic2752.xml

```diff
@@ -176,40 +181,26 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI332971">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    σῶμα μὲν Εὐστοχί
-                    <unclear>ο</unclear>
-                    <supplied reason="lost">υ τοῦ — —</supplied>-<lb n="2"/>%⁸⁰%⁸⁰νος ἐν χθονὶ κεῖ
-                    <unclear>τ</unclear>
-                    <supplied reason="lost">αι</supplied>, |<lb n="3"/>ὃς λίπε<unclear>ν</unclear> ἐν Λυκίῃ νή
-                    <unclear>π</unclear>
-                    <supplied reason="lost">ια τέκνα ⏑ –</supplied>
-                    |<lb n="4"/>vvv κάλλιπε δ’ αὐλ
-                    <supplied reason="lost">ήν, – ⏑ ⏑</supplied>
-                    <lb n="5"/>κοιρανίην ἀρετ
-                    <supplied reason="lost">ῆς(?) –· | τόν</supplied>
-                    <lb n="6"/>vvv τ’ ἐπέδη
-                    <unclear>σ</unclear>
-                    <supplied reason="lost">ε κακὴ μοῖρα(?)</supplied>
-                    <lb n="7"/>ἄστει τῷδ
-                    <supplied reason="lost">ε | — — — — — — — —</supplied>
-                    <lb n="8"/>v ὀλεσ
-                    <unclear>η</unclear>
-                    <supplied reason="lost">— — — — — — — — —</supplied>
-                    <lb n="9"/>ψυχ
-                    <unclear>ὴ</unclear>
-                    <supplied reason="lost">— — — — — — — — — — —</supplied>
-                    <lb n="10"/>
-                    <supplied reason="lost">— — — — — — — — — — — — —</supplied>
-                    <lb n="11"/>
+                    <lb n="1"/>σῶμα μὲν Εὐστοχί<unclear>ο</unclear><supplied reason="lost">υ</supplied> <supplied reason="lost">τοῦ</supplied><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="2" break="no"/><space extent="unknown" unit="character"/>νος ἐν χθονὶ κεῖ<unclear>τ</unclear><supplied reason="lost">αι</supplied>
+                    <lb n="3"/>ὃς λίπε<unclear>ν</unclear> ἐν Λυκίῃ νή<unclear>π</unclear><supplied reason="lost">ια</supplied> <supplied reason="lost">τέκνα</supplied> <supplied reason="lost">⏑ –</supplied>
```

Comment:
Metrical fragments go in a combination of `seg` and `gap`, e.g.: `<seg met="-+"><gap reason="lost" quantity="4" unit="character" precision="low"/></seg>`, so we can capture both the suggested gap extent and the metrical reconstruction.